Makefile

```diff
@@ -37,4 +37,10 @@ clean:
 all: check test clippy fmt build doc
 
 .PHONY: ci-local
-ci-local: all
\ No newline at end of file
+ci-local: all
+
+bindgen:
+	@echo "Running bindgen..."
+	@cargo install cbindgen
```

Comment:
This seems like a code smell to me... installing packages should be a part of the runner setup in CI, and a documented manual step for local development, since it only needs to be done once per machine.